templates/cms/media.html

```diff
@@ -1,13 +1,13 @@
 {% extends "base.html" %}
 {% load static %}
 
-{% block headtitle %}{{media_object.title}} - {{PORTAL_NAME}}{% endblock headtitle %}
+{% block headtitle %}{{media_object.title|add:" - "|add:PORTAL_NAME|truncatechars:64}}{% endblock headtitle %}
 
 {% block headermeta %}
 
 <link rel="canonical" href="{{FRONTEND_HOST}}{{media_object.get_absolute_url}}">
 
-<meta name="description" content="{% if media_object.summary %}{{media_object.summary}}{% else %}{{media_object.description}}{% endif %}">
+<meta name="description" content="{% if media_object.summary %}{{media_object.summary|truncatechars:160}}{% else %}{{media_object.description|truncatechars:160}}{% endif %}">

```

Comment:
consider doing the same for the `meta property="og:description"` too ?

Reply:
Added as a commit with a limit of 300, since there doesn't seem to be a hard limit on that like there is on meta description tags but 300 seems to be a commonly-agreed-upon maximum length for that.